### ISSUE TITLE
Adventures show scores

### ISSUE THREAD

**Issue by Arcadius2006** [LBPUnion/ProjectLighthouse#720]
**Describe the bug**
Adventures show scores on the website

**To Reproduce**
1. Go to the Levels page
2. Locate and click an adventure (they have hexagonal icons)
3. Notice that there is a score tab

**Expected behavior**
No score tab is present

**Screenshots**

**Environment Details**
 - OS: [e.g. iOS 15, Windows 10]
 - Browser and Version [e.g. Google Chrome v91, Safari v15]

**Additional context**
LBP3 doesn't show scores on adventures, so why should the website?

**Comment by W0lf4llo** [comment]
Yeah, the levels themselves are supposed to have the top scores, maybe we could add the actual sub levels of the adventure at the bottom that we can click on and view the leaderboards

**Comment by Arcadius2006** [comment]
Whoops! i actually forgot about that, i genuinely thought you could only see the scores in-level

i shall edit my issue accordingly
